src/app.controller.ts

```diff
@@ -185,10 +188,21 @@ export class AppController {
       throw new HttpException('Data startu nie może być po dacie zakończenia wydarzenia.', HttpStatus.BAD_REQUEST);
 }}
 
-async assert_voting_end(voting_date: Date){
+  async assert_voting_end(voting_date: Date){
   const current_date = new Date();
   if (current_date > voting_date) {
     throw new HttpException('Nieprawidłowo podano datę zakończenia głosowania', HttpStatus.BAD_REQUEST);
-}}
+  }}
+
+  async assert_is_voting_open(event_id: string) {
+  const event = await this.prisma.event.findUnique({
+    where: { event_id: event_id },
+    select: { voting_end: true }
+  });
+  const current_date = new Date();
+  if (current_date > event.voting_end) {
+    throw new HttpException('Głosowanie na to wydarzenie zostało zakończone.', HttpStatus.BAD_REQUEST);
+    }
+  }
```

Comment:
w evencie oprócz daty można sprawdzić również pole które odpowiada za to czy event jest zakończony